creme/preprocessing/scale.py

```diff
@@ -84,3 +84,81 @@ def transform_one(self, x):
             i: (xi - self.variances[i].mean.get()) / (self.variances[i].get() + self.eps) ** 0.5
             for i, xi in x.items()
         }
+
+class MinMaxScaler(base.Transformer):
+    """Scales the data to a fixed range 0 to 1.
+
+    Under the hood a running min and a running peak to peak (max - min) are maintained. The scaling is slightly
+    different than when using scikit-learn but this doesn't seem to have any impact learning
+    performance.
+
+    Attributes:
+        min (dict): Mapping between features and instances of ``stats.Min``.
+        p2p (dict): Mapping between features and instances of ``stats.PeakToPeak``.
+        eps (float): Used for avoiding divisions by zero.
+
+    Example:
+
+        >>> import pprint
+        >>> import creme
+        >>> import numpy as np
+        >>> from sklearn import preprocessing
+        
+        >>> rng = np.random.RandomState(42)
+        >>> X = [{'x': v} for v in rng.uniform(low=8, high=12, size=15)]
+        
+        >>> scaler = creme.preprocessing.MinMaxScaler()
+        >>> pprint.pprint([scaler.fit_one(x) for x in X])
+        [{'x': 0.0},
+         {'x': 1.0},
+         {'x': 0.620391...},
+         {'x': 0.388976...},
+         {'x': 0.0},
+         {'x': 0.0},
+         {'x': 0.0},
+         {'x': 0.905293...},
+         {'x': 0.608349...},
+         {'x': 0.728172...},
+         {'x': 0.0},
+         {'x': 1.0},
+         {'x': 0.855194...},
+         {'x': 0.201990...},
+         {'x': 0.169847...}]
+        
+        >>> X = np.array([x['x'] for x in X]).reshape(-1, 1)
+        >>> preprocessing.MinMaxScaler().fit_transform(X)
+        array([[0.37284965],
+               [0.9797798 ],
+               [0.74938422],
+               [0.60893137],
+               [0.14266357],
+               [0.14263816],
+               [0.03950081],
+               [0.89072903],
+               [0.61151903],
+               [0.72418595],
+               [0.        ],
+               [1.        ],
+               [0.85519484],
+               [0.20199041],
+               [0.16984743]])
+
+    """
+    
+    def __init__(self):
+        self.min = collections.defaultdict(lambda : stats.Min())
+        self.p2p = collections.defaultdict(lambda : stats.PeakToPeak())
```

Comment:
I think you should use a `max`. The problem is that by using a `p2p` you're computing the `min` twice.